p/themes/Origine/origine.css

```diff
@@ -911,14 +901,14 @@ a.btn {
 
 /*=== Notification and actualize notification */
 .notification {
-	padding: 0 0 0 5px;
+	padding: 15px;
 	font-size: 0.9em;
 	border: 1px solid #eeb;
 	border-radius: 3px;
 	box-shadow: 0 0 5px #ddd;
 	text-align: center;
 	font-weight: bold;
-	line-height: 3em;
+	line-height: 1.3;
```

Comment:
Is that necessary at all? Default is usually between 1.2 and 1.3.

Reply:
it is necessary. in base-theme/base.css the line-height is 3em
![grafik](https://user-images.githubusercontent.com/1645099/129787548-d8c2b519-5aac-4177-a39b-d6eb0b903f38.png)

Reply:
That would mean `line-height: **3**`, not 1.3. This line should be removed because 3 is a mistake and 1.3 is useless.

Reply:
ah ok. I got the point now.
honestly: I personally do not like line-height: 3em in a notification. The gap between 2 lines of a notification is huge (especially in mobile view).

I will delete the line...